R/reexports.R

```diff
@@ -0,0 +1,56 @@
+# Re-exports
```

Comment:
`reexports.R`

Re-export essential functions from other packages so that when `proActiv` is loaded, these functions are accessible via proActiv namespace without having to load the other packages